### PR TITLE
Confirmation email to customer after submit

### PR DESCRIPTION
**Pull request by oliverochman**
PT https://www.pivotaltracker.com/story/show/178566944

### User Story
```
As Flex Coast
To provide quality customer service
We want to send a confirmation email to our customers of their inquiry
```

### Changes proposed
- When an inquiry is submitted an email is sent to the creator of it

### CODE REVIEW

**Review by lukeperrin10 — APPROVED** [review]

File: app/views/inquiry_mailer/submitter_email.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>We have you covered! You are looking for an office for <%=Inquiry.last.size%> people and we will find one for you. Expect to hear from us within a day or two...</h1>
```
Comment (offcast-dev) [inline app/views/inquiry_mailer/submitter_email.html.erb]: not critical, but if it is office for 1 person grammar wont work. I say either we have if statement for that, or say something like:
We received your inquiry and we have you covered! Our team will select the best offices for you, expect to hear from us withing a day or two. Meanwhile fell free to contact us, our phone is 031-123-4567

**Review by offcast-dev — COMMENTED** [review]
not sure about text in email

**Review by DavDan1 — APPROVED** [review]

File: app/views/inquiry_mailer/submitter_email.html.erb (hunk 1 above)
Comment (oliverochman) [reply app/views/inquiry_mailer/submitter_email.html.erb]: addressed

File: app/views/inquiry_mailer/submitter_email.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>We received your inquiry and we have you covered! Our team will select the best offices for you, expect to hear from us withing a day or two. Meanwhile fell free to contact us, our phone is 031-123-4567</h1>
+  </body>
+</html>
```
Comment (1CIM) [inline app/views/inquiry_mailer/submitter_email.html.erb:8]: Should we have this in Swedish as well?

**Review by LucasKnudsen — APPROVED** [review]

**Pull request merged**